mailbox/caching/pom.xml

```diff
@@ -54,6 +54,16 @@
             <artifactId>junit</artifactId>
```

Comment:
Replace this dependency with junit-vintage

Reply:
I think I can just delete it in fact !